pkg/benchmarkjunit/README.md

```diff
@@ -1,8 +1,9 @@
 # benchmarkjunit
-
 `benchmarkjunit` is a go utility that runs go [`testing` package](https://golang.org/pkg/testing/#hdr-Benchmarks) benchmarks with a command like `go test -v -bench=. <packages>` and translates the output into JUnit XML.
 
 ## Usage Examples
+Run `go run ./pkg/benchmarkjunit --help` to see all the available flags.
```

Comment:
Don't we want to nudge people in the direction of bazel?

Reply:
I favored using just go instead of bazel since this is a utility for processing go benchmarks and the `golang` image should be sufficient for all use cases if  `go run` is used instead of `bazel run`.
I'd like to keep the examples of the pure golang approach since a lot of projects don't use bazel, but I can also add a usage example that uses bazel.

Reply:
Added a bazel example.